src/app/components/Staking/Staking.tsx

```diff
@@ -366,10 +366,15 @@ export const Staking: React.FC<StakingProps> = ({
         );
         return stakingFeeSat;
       } catch (error: Error | any) {
+        let errorMsg = error?.message;
+        if (error?.message.includes("Insufficient funds")) {
```

Comment:
Lets add a code that we enhance this error message so it's easier to navigate later on